tests/foreman/cli/test_host.py

```diff
@@ -706,14 +705,14 @@ def test_positive_list_infrastructure_hosts(
     target_sat.cli.Host.update({'name': target_sat.hostname, 'new-organization-id': module_org.id})
     # list satellite hosts
     hosts = target_sat.cli.Host.list({'search': 'infrastructure_facet.foreman=true'})
-    assert len(hosts) == 2 if is_open('BZ:1994685') else len(hosts) == 1
+    assert len(hosts) == 1
```

Comment:
as 1994685 was closed wontfix, so maybe we should keep 2. On the other hand, the issue might not persist, so this warrants a local run

Reply:
It doesn't matter as so far the code was checking the else part `len(hosts) == 1` as BZ: 1994685 has been closed(no matter the kind of resolution) for more than 5 months.

Reply:
hm, it does matter if it passes or fails, digging through past runs 
![image](https://github.com/user-attachments/assets/7703c875-fbfc-4d8c-9c46-78ffda1c02b0)
so if you could flip it to 2 in both cases I'd be thankful

Reply:
Updated the PR